controllers/bounce_processes.go

```diff
@@ -43,6 +41,10 @@ type bounceProcesses struct{}
 
 // reconcile runs the reconciler's work.
 func (bounceProcesses) reconcile(ctx context.Context, r *FoundationDBClusterReconciler, cluster *fdbv1beta2.FoundationDBCluster) *requeue {
+	if !pointer.BoolDeref(cluster.Spec.AutomationOptions.KillProcesses, true) {
+		return nil
```

Comment:
You don't want to record the old message here? It used to add an event with "Spec require a bounce of some processes, but killing processes is disabled"  and log an error?

Reply:
Good, point. I'll add that back.